R/indexing.R

```diff
@@ -391,37 +389,31 @@ length.igraph <- function(x) {
     (is.null(attr) && is.numeric(value) && value == 0)) {
     ## Delete edges
     if (missing(i) && missing(j)) {
-      todel <- unlist(x[[, , ..., edges = TRUE]])
+      todel <- seq_len(ecount(x))
```

Comment:
This looks consistent with the current implementation, but do we really expect all edges to be deleted with `g[] <- ...` ?

Reply:
I think it makes sense that `g[]<-0` deletes all edges. If it is useful is another question but at least it is consistent.

Reply:
Oh, right. Makes sense.